KhronosGroup/glTF-Blender-IO
# Export named vertex attribute sets as custom attributes.

Proposed fix for https://github.com/KhronosGroup/glTF-Blender-IO/issues/542.

Not ready to merge – there's one TODO in the code, and I'd like to get more feedback on whether this is the right approach and whether it's useful.

Steps to test:

1. Create one or more vertex color sets.
2. Assign a custom (non-"Col") name to at least one vertex color set.
3. Export with the "Meshes -> Custom Attributes" option _enabled_.
4. Result should be something like this, given a vertex color set named "Flux":

![Screen Shot 2019-11-29 at 10 49 32 PM](https://user-images.githubusercontent.com/1848368/69896991-79095480-12fa-11ea-97c9-e06947c480d7.png)


![Screen Shot 2019-11-29 at 10 45 06 PM](https://user-images.githubusercontent.com/1848368/69896949-dbae2080-12f9-11ea-8130-1c767d19bb0e.png)


This will import to three.js with the name `mesh.geometry.attributes._flux`, for example.
